src/main/java/seedu/duke/helper/UI.java

```diff
@@ -39,10 +39,28 @@ public void printGreeting() {
     public void printHelp() {
         printNewLineSeparator();
         System.out.println("Here are the commands and examples:");
-        System.out.println("1. add patient\n2. delete patient\n3. view patient");
-        System.out.println("4. add doctor\n5. delete doctor\n6. view doctor");
-        System.out.println("6. add patient\n7. delete patient\n9. view patient");
-        System.out.println("10. bye");
+        System.out.println("1. add patient /info. Example:");
+        System.out.println("add patient /info S1234567A, John Doe, 23, M, 10 Baker Street," +
+                " 1999-12-31, 2021-02-15");
+        System.out.println("2. delete patient. Example:");
+        System.out.println("delete patient /info 1");
+        System.out.println("3. view patient. Example:");
+        System.out.println("view patient");
+        System.out.println("4. add doctor. Example:");
+        System.out.println("add doctor /info S1234567A, Shirley Tan, 40, F, 1 Baker Road, " +
+                "1980-12-31, Dermatology");
+        System.out.println("5. delete doctor. Example:");
+        System.out.println("delete doctor /info 1");
+        System.out.println("6. view doctor. Example:");
+        System.out.println("view doctor");
+        System.out.println("7. add medicine. Example:");
+        System.out.println("add medicine /info 1, Paracetamol, 500, 2023-06-11, Slight headache, 10");
```

Comment:
medicine the user doesnt have to give the initial id, the method will auto assign

Reply:
alright will remove "1," from add medicine